test/simplified/five-spec.k

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019 K Team. All Rights Reserved.
```

Comment:
For each of these specs, could you give a brief description of what it is testing? It's not clear to me what the failure modes are.